prime-router/docs/schema_documentation/az-az-covid-19.md

```diff
@@ -1,11 +1,17 @@
 
 ### Schema:         az/az-covid-19
+### Topic:          covid-19
+### Tracking Element: null (testing_lab_specimen_id)
+### Base On: [covid-19](./covid-19.md)
+### Extends: [none](./none.md)
```

Comment:
Add code to not have this link when there is no value.

Reply:
Done.